app/views/admin/action_items/_rejected_auctions.html.erb

```diff
@@ -15,7 +15,7 @@
       <td><%= link_to(auction.title, admin_auction_path(auction.id)) %></td>
       <td><%= auction.delivery_due_at %></td>
       <td><%= auction.delivery_url %></td>
-      <td><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %></td>
+      <td><% if auction.winning_bidder? %><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %><% end %></td>
```

Comment:
Yeah, I thought about that, but it means basically calling `html_safe` on something from the presenter and putting view model stuff deep in the NullBid object so I didn't think that would be better...